e2e/starter.test.js

```diff
@@ -0,0 +1,13 @@
+describe('Example', () => {
```

Comment:
Here try this change:

```suggestion
import { device, element, by } from 'detox';

describe('Example', () => {
```